api/swagger/swagger.yaml

```diff
@@ -248,7 +248,93 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
+  /stats:
+    x-swagger-router-controller: stats
+    get:
+      tags:
+        - statistics
+      description: Returns statistics
+      operationId: get
+      responses:
+        "200":
+          description: Success
+          schema:
+            $ref: "#/definitions/Stats"
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+
 definitions:
+  Stats:
+    type: object
+    properties:
+      trialsCount:
+        type: integer
+        description: Trials count
+      trialsPerRegistry:
+        type: array
+        items:
+          $ref: '#/definitions/TrialsPerRegistry'
+      trialsPerYear:
+        type: array
+        items:
+          $ref: '#/definitions/TrialsPerYear'
+      topLocations:
+        type: array
+        items:
+          $ref: '#/definitions/TopLocations'
+      dateRegistry:
```

Comment:
Please rename this to `sourcesLatestUpdatedDate`. Rename also `latest_update_date` to `latest_updated_date`. Sorry I'm being picky here, but as these attributes are part of our public API, it's difficult to change names once we pick one.